frontend/src/metabase/components/MetadataInfo/DimensionInfoPopover/DimensionInfoPopover.tsx

```diff
@@ -22,17 +23,33 @@ type Props = { dimension: Dimension } & Pick<
   "children" | "placement"
 >;
 
+const className = "dimension-info-popover";
+
 function DimensionInfoPopover({ dimension, children, placement }: Props) {
   // avoid a scenario where we may have a Dimension instance but not enough metadata
   // to even show a display name (probably indicative of a bug)
   const hasMetadata = !!(dimension && dimension.displayName());
 
   return hasMetadata ? (
     <TippyPopver
```

Comment:
Not related to your changes, but this component should be renamed. `TippyPopover` instead of `TippyPopver`.

Reply:
oops, apparently i'm blind -- will fix in a separate pr as there is also a type in `TableInfoPopover`. thanks for catching.